### PR TITLE
Feat/client registration

### PR DESCRIPTION
**Pull request by Karonuk**

### CODE REVIEW

File: E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:E_Hospital.Client"
+        xmlns:Controls="clr-namespace:MahApps.Metro.Controls;assembly=MahApps.Metro"
+        mc:Ignorable="d"
+        Title="PatientRegistration" Height="350" Width="450">
+    <Grid>
+        <StackPanel Name="RegisterPanel" VerticalAlignment="Center"  HorizontalAlignment="Center" Width="400">
```
Comment (lemichello) [inline E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml]: Change Width to 200

File: E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                return true;
+            else
+                return false;
+        }
+        private void RegistrationClick(object sender, RoutedEventArgs e)
+        {
+            if (CheckForNull())
+            {
+                NewPatient.Password = ClientPasswordBox.Password;
+                NewPatient.Role = RegistrationService.UserRoles.Patient;
```
Comment (lemichello) [inline E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml.cs:52]: Where is assigning first name and last name?

File: E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    /// </summary>
+    public partial class PatientRegistrationWindow : MetroWindow
+    {
+        public RegistrationService.PatientDto NewPatient { get; set; }
+        public PatientRegistrationWindow()
+        {
+            NewPatient = new RegistrationService.PatientDto();
+            _client = new RegistrationService.RegistrationServiceClient();
+            InitializeComponent();
+            _client.Open();
```
Comment (lemichello) [inline E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml.cs:34]: For what? Delete.

File: E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    /// <summary>
+    /// Interaction logic for ClientRegistration.xaml
+    /// </summary>
+    public partial class PatientRegistrationWindow : MetroWindow
+    {
+        public RegistrationService.PatientDto NewPatient { get; set; }
+        public PatientRegistrationWindow()
+        {
+            NewPatient = new RegistrationService.PatientDto();
+            _client = new RegistrationService.RegistrationServiceClient();
```
Comment (lemichello) [inline E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml.cs:32]: Initialize fields after `InitializeComponent` method execution.

File: E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml.cs (hunk 2 above)
Comment (Karonuk) [reply E-Hospital/E-Hospital.Client/PatientRegistrationWindow.xaml.cs:52]: Binding in xaml

**Review by lemichello — APPROVED** [review]

**Pull request merged**